src/utils.js

```diff
@@ -1,6 +1,9 @@
 import url from 'url';
-
 import fs from 'fs';
+import path from 'path';
+
+const __filename = url.fileURLToPath(import.meta.url);
```

Comment:
I suspect these could be removed (see below). If not, `__` is usually a sign of some sort of internal platform value and shouldn't be used for custom variables. Alternative:

```js
const PROJECT_ROOT_DIR = fileURLToPath(new URL('../../', import.meta.url));
```

Reply:
After the above modifications, this was not necessary, but I'll keep this alternative in mind if necessary in #17